### ISSUE TITLE
DAO Dashboard - Token Listing

### ISSUE THREAD

**Issue by jessicazartler** [TECommons/Communications-Working-Group#11]
## 📋   Anything to add?
https://deepdao.io/#/deepdao/dashboard

## 🎉   Subtasks

- [ ] 
- [ ] 
- [ ] 

## 🤼‍♂️   Reviewer

@

**Comment by tamarandom** [comment]
Hey @jessicazartler We mean once the TEC token is live, right? So after the Commons Upgrade.

**Comment by jessicazartler** [comment]
Yes, after the token is live, this is a place to list :D Thanks Tam!

**Comment by chuygarcia92** [comment]
Already sent an email also trying to list Gardens as a DAO platform. @0xGabi mentioned they are already in contact with them at 1Hive and it should be a low-hanging fruit :apple:

**Comment by 0xGabi** [comment]
We started the communication with the DeepDAO team and we are working on a document to explain how to request the data for Gardens.

**Comment by chuygarcia92** [comment]
This is the WIP document: https://hackmd.io/@1sBtuDmrRBG360k8-EXf2A/Hy2KqfJSY/edit

**Comment by chuygarcia92** [comment]
@0xGabi and I had a call with the DeepDAO team. On standby for Aragon Connect issues, but gave them updates on token data and the supply API.
